### PR TITLE
Allow fluentd env vars, useful for plugins

### PR DESCRIPTION
**Pull request by PabloCastellano**

### CODE REVIEW

File: log-router/values.yaml [hunk 1]
```diff
@@ -46,7 +46,9 @@ labelSelector:
 
 
 fluentd:
+  # Use extraEnv to define env vars from secrets and extraPluginsEnv to define env vars directly
```
Comment (jvassev) [inline log-router/values.yaml]: Nice touch here! Can you include this doc string in the README too?

File: log-router/templates/service.yaml [hunk 2]
```diff
@@ -15,6 +15,7 @@ metadata:
     prometheus.io/scrape: "true"
     prometheus.io/port: "24231"
   name: {{ template "fluentd-router.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```
Comment (jvassev) [inline log-router/templates/service.yaml]: Not sure it is needed. I think helm injects the namespace - none of the stable charts define it, for example https://github.com/helm/charts/blob/master/stable/kubernetes-dashboard/templates/deployment.yaml

Comment (PabloCastellano) [reply log-router/templates/service.yaml]: Oh, I got confused and you are right. Then I'm removing this part of the PR

**Comment by PabloCastellano** [comment]
Hi. Is there anything left to merge this PR? :smiley:

**Comment by jvassev** [comment]
Hi @PabloCastellano,

Sorry for my late reply. Now that I look at it, we can get away without this feature. The `fluentd.extraEnv` also serves your use case and has the added benefit of securing your secrets by putting them in a `Secret`. So the fluentd container will get the secrets but they will not be stored in the  pod spec but in a secret.

**Pull request closed without merging**